### PR TITLE
Showing status after enabling/disabling autoupdate

### PR DESCRIPTION
**Pull request by castulo**
Signed-off-by: Castulo Martinez <castulo.martinez@intel.com>

### CODE REVIEW

File: src/autoupdate.c [hunk 1]
```diff
@@ -135,6 +135,7 @@ enum swupd_code autoupdate_main(int argc, char **argv)
 		 */
 		if (globals.path_prefix) {
 			warn("Running autoupdate with --path will not restart swupd-update.timer. This will have to be done manually\n");
+			info("Autoupdate enabled\n");
```
Comment (otaviobp) [inline src/autoupdate.c:138]: 2 things here. First I would use the same wording used when no parameter is used. And also I think we should to run the same systemctl commands to make sure that the autoupdate is really enabled.

Comment (otaviobp) [reply src/autoupdate.c:138]: The only problem is that if a restart is needed we would print something like
```
Running systemctl to enable updates
Disabled
```

Not sure if it's clear enough

Comment (castulo) [reply src/autoupdate.c:138]: > 2 things here. First I would use the same wording used when no parameter is used. And also I think we should to run the same systemctl commands to make sure that the autoupdate is really enabled.

Sure, I can change this

Comment (castulo) [reply src/autoupdate.c:138]: > The only problem is that if a restart is needed we would print something like
> 
> ```
> Running systemctl to enable updates
> Disabled
> ```
> 
> Not sure if it's clear enough

I do think this is very confusing

Comment (otaviobp) [reply src/autoupdate.c:138]: So maybe we should let the way it is or find a better way to report this

**Comment by castulo** [comment]
Canceling this PR until we come up with a better solution

**Pull request closed without merging**